app/controllers/articles_controller.rb

```diff
@@ -12,7 +16,7 @@ def results
   end
 
   def show
-    unless @article
+    if @article.nil?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.